src/test/java/com/springweb/dv_spring_web_mongo/ProjectServiceTest.java

```diff
@@ -126,16 +123,16 @@ public void getProjectByIdTest() {
         });
     }
 
-    @Test
-    public void addNewProjectTest() {
-        //given
-        projectRepository.deleteAll();
-        projectService.addNewProject(new ProjectCreateOrUpdateDTO(testProject.getProjectName()));
-        //when
-        Project actual = projectRepository.findProjectByProjectName(testProject.getProjectName());
-        //then
-        Assertions.assertEquals(testProject.getProjectName(), actual.getProjectName());
-    }
+//    @Test
```

Comment:
what is the reason to comment out the method? You can add @Ignore annotation if the test is failed, but it is better to fix all problems